configs/exec/default.json

```diff
@@ -0,0 +1,10 @@
+{
+  "max-run"      : 30,
+  "max-retry"    : 15,
+  "advance"      : false,
+  "keep-browser" : false,
+  "comment"      : "<today>",
+  "user-simulation-tool" : "sikuli",
```

Comment:
I still think that this should not be an implication for "there is another user-simulation-tool, but no example for it".